ebpf/throughput/throughput.c

```diff
@@ -36,6 +37,11 @@ int packet_classifier(struct __sk_buff *skb) {
 	void *data = (void*)(long)skb->data;
 	void *data_end = (void*)(long)skb->data_end;
 
+	__u8 is_ingress = 0;
+	if (skb->ifindex == skb->ingress_ifindex) {
+		is_ingress = 1;
+	}
```

Comment:
These two fields in the sk_buff struct are what allow us to determine during runtime whether this program is running on the ingress or egress qdisc.